engine/validation/fields.php

```diff
@@ -402,6 +402,16 @@ function mime(array $mimes)
         return $this;
     }
 
+    function max_size(int $size)
```

Comment:
`size` is unused here

Reply:
You can resolve if you don't want to fix, and I'll approve since it's very minor :)